### PR TITLE
Cleanup Gdi32 Ext/Org interop

### PR DESCRIPTION
**Pull request by hughbe**
## Proposed Changes
- Cleanup SetWindowExtEx/SetWindowOrgEx
- Cleanup GetViewportOrgEx/SetViewportOrgEx
- Cleanup GetViewportExtEx/SetViewportExtEx

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/2130)

### CODE REVIEW

File: src/Common/src/Interop/Gdi32/Interop.GetViewportExtEx.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+using System.Drawing;
+using System.Runtime.InteropServices;
+using System.Windows.Forms;
+
+internal static partial class Interop
+{
+    internal static partial class Gdi32
+    {
+        [DllImport(Libraries.Gdi32, ExactSpelling = true)]
+        public static extern BOOL GetViewportExtEx(IntPtr hdc, out Size lpsize);
```
Comment (zsd4yr) [inline src/Common/src/Interop/Gdi32/Interop.GetViewportExtEx.cs]: If folks are intended to use the wrapper function, should this extern be made private?

Comment (hughbe) [reply src/Common/src/Interop/Gdi32/Interop.GetViewportExtEx.cs]: It isn't intended to be private. Some callers pass `IntPtr`, others pass `IHandle`

Comment (zsd4yr) [reply src/Common/src/Interop/Gdi32/Interop.GetViewportExtEx.cs]: Thanks for the clarity :)

**Comment by zsd4yr** [comment]
@msftbot please merge if @RussKie approves

**Pull request merged**